zcash_client_sqlite/src/wallet/common.rs

```diff
@@ -251,7 +260,7 @@ where
                    JOIN transactions stx ON stx.id_tx = transaction_id
                    WHERE stx.block IS NOT NULL -- the spending tx is mined
                    OR stx.expiry_height IS NULL -- the spending tx will not expire
-                   OR stx.expiry_height > :anchor_height -- the spending tx is unexpired
+                   OR stx.expiry_height >= :target_height -- the spending tx is unexpired
```

Comment:
This is correct; the transaction is unexpired if its expiry is the target height, because it can still be mined at the target height.